changelog/differ_test.go

```diff
@@ -122,7 +124,7 @@ func TestCompare(t *testing.T) {
 	for _, opt := range tests {
 		t.Run(opt.name, func(t *testing.T) {
 			got, err := diffItems(opt.old, opt.new)
-			assert.NoError(t, err)
+			require.NoError(t, err)
```

Comment:
Do we need to exit on the first error?

Reply:
[The same](https://github.com/aiven/terraform-provider-aiven/pull/2193#discussion_r2075177418)